aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -67,23 +78,40 @@ sub get_anaconda_version
     return $version;
 }
 
-# Returns the absolute path where the PXE file must be written.
+# Prints an IP address in hexadecimal.
+sub hexip
```

Comment:
Why, this will make the code more complex with a lot of duplication. Just the format method is very different (the syntax of the file is not at all the same and there it would make the function much more complex to have one function with a test of the variant... At least this is my feeling!